infra/image/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -eu
```

Comment:
really minor: what about "#!/bin/bash -euo pipefail" it ensures the script exits on errors, the unset vars as errors, piped commands...